### PR TITLE
feat: support new application locale `pt-BR`

### PR DESCRIPTION
**Pull request by emmenko**

### CODE REVIEW

File: packages/i18n/data/pt-BR.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (emmenko) [inline packages/i18n/data/pt-BR.json:1]: Empty for now until the translations in TX are ready

File: transifex.yml
Comment (emmenko) [inline transifex.yml:1]: Better to use the config file so it's easier and more transparent to update.

I'll update the integration to point to this file once the PR is merged.

File: transifex.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  filters:
+    - filter_type: file
+      file_format: KEYVALUEJSON
+      source_language: en_US
+      source_file: packages/i18n/data/core.json
+      translation_files_expression: packages/i18n/data/<lang>.json
+  settings:
+    language_mapping:
+      fr_FR: fr-FR
+      pt_BR: pt-BR
```
Comment (emmenko) [inline transifex.yml:11]: Here's the mapping for the new locale

**Review by CarlosCortizasCT — APPROVED** [review]
Thanks! 👍

**Review by ddouglasz — APPROVED** [review]
Thank you 👍🏾

**Review by Rhotimee — APPROVED** [review]

File: packages/calendar-time-utils/src/formats.ts [hunk 3]
```diff
@@ -50,6 +50,10 @@ export const DATE_FORMAT_LOCALIZED_MAPPINGS: TLocalizedDateFormatMapping = {
     [formatToken.DD]: 'JJ',
     [formatToken.YYYY]: 'AAAA',
   },
+  pt: {
+    // DD/MM/YYYY - HH:mm -> DD/MM/AAAA - HH:mm
+    [formatToken.YYYY]: 'AAAA',
+  },
```
[the remark this answers is not in the document]
Comment (emmenko) [reply packages/calendar-time-utils/src/formats.ts:56]: Ok, I'll try to find a reliable way to get this information and include it in the follow up documentation.

**Pull request merged**